.markdownlint.json

```diff
@@ -5,6 +5,7 @@
     "siblings_only": true
   },
   "line-length": false,
+  "link-image-reference-definitions": false,
```

Comment:
This new rules is incompatible with custom header slugs in RFCs, e.g.:

```md
# Summary

[summary]: #summary

One paragraph explanation of the feature.
```

Playground: https://dlaa.me/markdownlint/